website/src/community/contact-us.md

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 # Contact Us
 
 There are many ways to reach the Beam user and developer communities - use
-whichever one seems best!
+whichever one seems best.
```

Comment:
Don't we need to encourage users to contact by user@ in the first order? Seems like this is the most responsive type of channels.